lib/data/external/AwsClient.js

```diff
@@ -187,13 +198,30 @@ class AwsClient {
         });
     }
 
-    createMPU(key, metaHeaders, bucketName, websiteRedirectHeader, log,
-    callback) {
+    createMPU(key, metaHeaders, bucketName, websiteRedirectHeader, contentType,
+        cacheControl, contentDisposition, contentEncoding, log, callback) {
+        const metaHeadersTrimmed = {};
+        Object.keys(metaHeaders).forEach(header => {
+            if (header.startsWith('x-amz-meta-')) {
+                const headerKey = header.substring(11);
+                metaHeadersTrimmed[headerKey] = metaHeaders[header];
+            }
+        });
+        if (metaHeadersTrimmed !== {}) {
```

Comment:
This needs to be changed to `Object.keys.length > 0` as this condition always evaluates to true.
Note: What's the downside of sending empty object? I don't think it will throw an error.

Reply:
You're right, I will just assign the object here to remove this conditional. If it's empty, that will work.

Reply:
Updated.